controllers/apply/lib/joi-extensions/word-count.js

```diff
@@ -3,49 +3,61 @@ const countWords = require('../count-words');
 
 module.exports = function wordCount(joi) {
     return {
+        type: 'string',
         base: joi.string(),
-        name: 'string',
-        language: {
-            maxWords: 'must have less than {{max}} words',
-            minWords: 'must have at least {{min}} words',
+        messages: {
+            'string.minWords': 'must have at least {{#min}} words',
+            'string.maxWords': 'must have no more than {{#max}} words',
```

Comment:
Why the wording change?

Reply:
The previous wording was incorrect, it said "less than" when it's really "up to". This is just an internal message but wanted to make sure it was accurate.